client/dev/sheet/sheet.jsx

```diff
@@ -0,0 +1,161 @@
+            score: 1,
+            fraction: 0,
+            id: 4
+          },
+          {
+            name: "Constitution",
+            score: 1,
+            fraction: 0,
+            id: 5
+          },
+          {
+            name: "Looks",
+            score: 1,
+            fraction: 0,
+            id: 6
+          },
+          {
+            name: "Charisma",
+            score: 1,
+            fraction: 0,
+            id: 7
+          },
+          
+        ],
+        quirks: [
+        { 
+          name: "Test",
+          id: 0
+        }
+        ]
+      }
+    }
+  },
+  
+  /* Set state */  
+  getInitialState: function() {
+    return {
+      character: { 
+        stats: this.props.initialStats
+      }
+    };
+  },
+  
+  /* Define functions */ 
+  onScoreChange: function( index, delta ){
+    var newScore = this.state.character.stats.abilities[index].score + delta;
+    if( newScore <= 25 && newScore >= 1 ){
+      this.state.character.stats.abilities[index].score = newScore;
+      this.setState( this.state );
+    }
+  },
+  
+  onQuirkAdd: function( name ){
+    this.state.character.stats.quirks.push({
+        name: name,
+        id: nextQuirksId,
+      });
+      this.setState(this.state);
+      nextQuirksId += 1;
+  },
+  
+  onQuirkDelete: function( index ){
+    this.state.character.stats.quirks.splice(index, 1);
+    this.setState(this.state);
+  },
+
+  /* Render */
+  render: function(){
+    return(
+      <div className="row">
+        <div className="col-sm-6 col-md-4">
+          <div className='panel panel-default'>
+            <div className='panel-heading'>Abilities</div>
+            <div className='panel-body'>
+              <table className='table table-responsive'>
+                <thead>
+                  <tr>
+                    <th>Ability</th>
+                    <th>Rating</th>
+                    <th>Fraction</th>
+                    <th>Modifier</th>
+                  </tr>
+                </thead>
+                <tbody>
+                {this.state.character.stats.abilities.map(function( stat, index ) {
+                  return (
+                    <AbilityScore
+                      onScoreChange={
+                        function(delta) {
+                          this.onScoreChange(index, delta)
+                        }.bind(this)
+                      }
+                      name={stat.name} 
+                      score={stat.score} 
+                      fraction={stat.fraction}
+                      key={stat.id} />
+                  );
+                }.bind(this))}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>        
+        <Quirks 
+          data={this.state.character.stats.quirks}
+          onQuirkAdd={
+            function( name ){
+              this.onQuirkAdd( name )
+              }.bind(this)
+          }
+          onQuirkDelete={
+            function( index ){ 
```

Comment:
Couldn't this just be this.onQuirkDelete.bind(this)? Why wrap it in the anonymous function if it takes in the same argument?